### PR TITLE
mocking neon-js calls to remove async issues

### PR DESCRIPTION
**Pull request by dvdschwrtz**
**What current issue(s) from Trello/Github does this address?**

I did not mock the async calls to neon-js when I originally wrote these tests, which is why I had async issues and needed try/ catch blocks and long setTimeouts. Now the 'neon-js' calls are mocked and it is working correctly.

**What problem does this PR solve?**

Testing the neon wallet

**How did you solve this problem?**

By adding in mock functionality for 'neon-js' calls

**How did you make sure your solution works?**

running 'npm test'

**Are there any special changes in the code that we should be aware of?**

there was one place I left a TODO comment where I put the same code for my axios mock adapter in 2 files. I was unable to make this DRY with my initial attempt and feel that this is an important PR to improve the current tests, so I created this PR and will come back to that minor issue later.

**Is there anything else we should know?**

- [ ] Unit tests written?

### CODE REVIEW

File: __tests__/components/WalletInfo.test.js [hunk 1]
```diff
@@ -137,16 +134,12 @@ describe('WalletInfo', () => {
       SET_CLAIM
     ];
     setTimeout(() => {
```
Comment (shawnmclean) [inline __tests__/components/WalletInfo.test.js]: Do we still need the setTimeout here?

File: __tests__/components/WalletInfo.test.js [hunk 2]
```diff
@@ -109,26 +117,15 @@ describe('WalletInfo', () => {
     ];
     deepWrapper.find('.refreshBalance').simulate('click');
     setTimeout(() => {
```
Comment (shawnmclean) [inline __tests__/components/WalletInfo.test.js]: Same here

File: __tests__/components/WalletInfo.test.js [hunk 3]
```diff
@@ -86,13 +98,9 @@ describe('WalletInfo', () => {
     expect(clipboard.writeText.mock.calls.length).toBe(0);
     deepWrapper.find('.copyKey').simulate('click');
     setTimeout(() => {
```
Comment (shawnmclean) [inline __tests__/components/WalletInfo.test.js]: Here too.

**Comment by dvdschwrtz** [comment]
@shawnmclean I took it out of the first one that checked copy to clipboard was called. I do need the setTimeouts for the other 2 places because the store.getActions() method needs that extra cycle in the event loop in order to return the correct results.

**Pull request merged**